client.go

```diff
@@ -243,6 +250,11 @@ func (uc Client) IsEnabled(feature string, options ...FeatureOption) (enabled bo
 			// TODO: warnOnce missingStrategy
 			continue
 		}
+
+		staticContext := &context.Context{AppName: uc.options.appName, Environment: "default"}
+
+		mergo.Merge(&opts.ctx, staticContext, mergo.WithOverride)
```

Comment:
I personally feel that it is a bit overkill to import a new package just to get this functionality. Mergo uses reflection internally to see which fields need to be merged and this has a runtime performance penalty. Since we are only working with a single struct of a well known type I think we can just implement our own function for this so we have full control of how contexts are merged now and in the future.

I started on this issue a few weeks ago and have an implementation ready (with a unit test) if you want to include, otherwise feel free to take a stab at it yourself.